.idea/misc.xml

```diff
@@ -8,4 +8,3 @@
     <output url="file://$PROJECT_DIR$/.idea/out" />
   </component>
 </project>
-
```

Comment:
This file still shows up in the aggregate patch as being modified (the file in your PR has a blank line removed vs the original).

Reply:
Sorry about that - should have checked.